app/models/miq_policy.rb

```diff
@@ -51,7 +51,7 @@ class MiqPolicy < ApplicationRecord
 
   attr_accessor :reserved
 
-  cattr_accessor :associations_to_get_policies
+  class_attribute :associations_to_get_policies
```

Comment:
The use of `self` To clarify the variable had me confused. Also the `<<` not seeming to work but `+=` did (for arrays). Could you explain the issue before and why we are not concerned with introducing bugs here?